creme/geolocation/populate.py

```diff
@@ -27,7 +27,10 @@
 from creme.creme_core.models import (
     BrickDetailviewLocation,
     BrickMypageLocation,
-    SettingValue,
+)
+from creme.creme_core.utils.settings import (
+    has_setting_value,
+    set_setting_value,
 )
```

Comment:
As you can see you have to replace 1 import by 2. And It hide to the new reader that It concerns a Creme's model, & not the settings.py for example. manager method would be far better.